src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSource.cs

```diff
@@ -639,64 +658,75 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
 
                 using (var asyncToken = _owner.OperationListener.BeginAsyncOperation("HasSuggestedActionsAsync"))
                 {
-                    // First, ensure that the snapshot we're being asked about is for an actual
-                    // roslyn document.  This can fail, for example, in projection scenarios where
-                    // we are called with a range snapshot that refers to the projection buffer
-                    // and not the actual roslyn code that is being projected into it.
-                    var document = range.Snapshot.GetOpenDocumentInCurrentContextWithChanges();
-                    if (document == null)
+                    var selection = await GetSpanAsync(range).ConfigureAwait(false);
+                    if (selection != null)
```

Comment:
It looks like this used to call HasFixes and HasRefactorings, even if the span was null.  Did we rely on that?